sys/pm_callouts.c

```diff
@@ -232,6 +198,38 @@ void redir_from_callout(pportmaster_packet_info packetInfo, pportmaster_packet_i
 
 }
 
+NTSTATUS inject_packet(pportmaster_packet_info packetInfo, int diraction, void *packet, ULONG packet_len) {
```

Comment:
Typo: `direction`

Reply:
I've not seen to code in a while, so I'm a bit confusion the `int direction`.
Can we change that to a `bool inbound` instead? I think this would make it a lot easier to read.
Or would this be part of the code refactor?